gui/create-item.lua

```diff
@@ -36,10 +36,10 @@ function getCreaturePartList(creatureID, casteID)
 end
 
 function getCreaturePartLayerList(creatureID, casteID, partID)
- local crplList={}
+ local crplList={{"whole"}}
  for k,crpl in ipairs(df.global.world.raws.creatures.all[creatureID].caste[casteID].body_info.body_parts[partID].layers) do
   local str = crpl.layer_name
-  table.insert(crplList,{str})
+  table.insert(crplList,{string.lower(str)}) -- lowercase string for easier searching
```

Comment:
instead of making these lower case, you could change the FilteredList to `case_sensitive=false`

Reply:
didn't know that, huh.

Reply:
actually how do i do that? just stick it in?